docker/docker-compose-hostdb.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.8"
+services:
+    app:
+      extends: 
+        file: docker-compose-generic.yml
+        service: app
+      image: ghcr.io/openboxes/openboxes:feature-upgrade-to-grails-3.3.10
+      # Use below 'image' property (with proper tag) if you use locally-built image
+      # image: openboxes/openboxes:latest
+      environment:
+        # It's hardcoded - app connects to the host's DB using host.docker.internal
+        DATASOURCE_URL: jdbc:mysql://host.docker.internal:3306/openboxes?useSSL=false
```

Comment:
This should be configurable.

Reply:
OK, I'll make this one default, and keep the DATASOURCE_URL configurable